sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/Config.scala

```diff
@@ -59,14 +59,15 @@ object Config {
     case class Fixed(at: Instant) extends Timestamps
   }
 
-  case class Cli(config: Config, output: URI)
+  case class Cli(config: Config, output: URI, region: Option[String])
```

Comment:
In my opinion it'd be better to put everything (so `output` and `region`)  in the config file rather than having a mixture of CLI args and config file. `region` should be a field of `output`.

Reply:
I thought about this, then decided not to change it. In retrospect I agree, the better way is all in the config.

In the interest of drawing a line under 0.4.0, I've made an issue to do it in next release: https://github.com/snowplow-incubator/snowplow-event-generator/issues/47